server/db/context/posts_context.ts

```diff
@@ -105,9 +105,21 @@ export const getPostInfo = async (post_id : number, user_id? : number) => {
             throw ServerError.notFound("존재하지 않는 게시글입니다.");
         }
 
-        return mapDBToPostInfo(rows[0]);
-        
+        let postInfo = mapDBToPostInfo(rows[0]);
+
+        if(postInfo.author_id !== user_id){
+            const sql2 = `UPDATE posts SET views = views + 1 WHERE id = ?`;
```

Comment:
updated_at을 현재 값으로 명시해서 updated_at 컬럼의 자동 변경을 막을 수 있습니다.
https://stackoverflow.com/questions/2844863/updating-entry-without-updating-timestamp

```suggestion
            const sql2 = `
                UPDATE posts
                SET views = views + 1,
                    updated_at = updated_at
                WHERE id = ?
            `;
```